docs/install/methods.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Installation Methods
+---
+
+RKE2 can be installed to a system in a number of ways, 2 of which being the preferred and supported methods. Those methods are tarball and RPM. Both of which are wrappers for the provided install script.
+
+### Tarball
+
+To install RKE2 via install you first need to get the install script. This can be done in a number of ways.
+
+This gets the script and immediately starts the install process.
+
+```bash
+curl -sfL https://get.rke2.io | sh -
+```
+
+This will download the install script and make it executable.
+
+```bash
+curl -sfL https://get.rke2.io --output install.sh
+chmod +x install.sh
+```
+
+#### Installation
+
+The install process defaults to tarball and the latest RKE2 version and no other qualifiers are necessary. However, if you want specify a version, you should set the `INSTALL_RKE2_CHANNEL` environment variable. An example below:
+
+```bash
+INSTALL_RKE2_CHANNEL=latest ./install.sh
+```
+
+When the install script is executed, it makes a determination of what type of system it is. If it's any form of a RedHat system, it will perform an RPM based installation, otherwise the script defaults to tarball. RPM based installation is covered below.
```

Comment:
```suggestion
When the install script is executed, it makes a determination of what type of system it is. If it's an OS that uses RPMs (such as CentOS or RHEL), it will perform an RPM based installation, otherwise the script defaults to tarball. RPM based installation is covered below.
```